src/app/Caches/Create/Features/IndexedCacheConfigurator.tsx

```diff
@@ -122,7 +132,13 @@ const IndexedCacheConfigurator = (props: {
 
   if (!props.isEnabled) {
     return (
-      <FeatureAlert feature={CacheFeature.INDEXED} />
+      <FeatureAlert feature={CacheFeature.INDEXED} errorType="types" />
```

Comment:
pass the error message directly with a property called error and display it in the FeatureAlert component instead of passing 2 extra properties